browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3548,6 +3548,18 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_VBAT_UNSUPPORTED_REGION_TEXT" desc="Vbat modal text for unsupported region">
       On <ph name="DUE_DATE">%1$s</ph>, we will be discontinuing support for existing virtual BAT balances. Unfortunately, there are no available custodians in your region (<ph name="REGION">%2$s</ph>) to withdraw your earnings. Until then, you can still contribute to your favorite creators.
     </message>
+    <message name="IDS_SPEEDREADER_MESSAGE_TITLE" desc="Message shown on the speedreader prompt.">
+      View page with Speedreader?
+    </message>
+    <message name="IDS_SPEEDREADER_MESSAGE_BUTTON" desc="The label for the button to enable speedreader on current page.">
+      Enable
+    </message>
+    <message name="IDS_ENABLE_SPEEDREADER_TITLE" desc="Title for preference that enables or disables Speedreader.">
+      Speedreader
```

Comment:
@rmcfadden3 Is `Speedreader` a translatable term?

Reply:
@mkarolin — I believe the answer is yes: We translate basically everything except for the word "Brave" itself, so we would also want to translate the word "speedreader" here.

But CC @laurentiugruia to confirm. He should have final say on that.

Reply:
Thank you for the tag, @rmcfadden3. 

Generally yes but we have exceptions sometimes, depending on the feature name and if translating it to the target language is tricky and/or affects the intended message.

In this case I've looked around in Transifex for quite a few languages and almost all translators kept it as "Speedreader", guessing due to the above reasoning.

I'm fine then with tagging it as "notranslate".

Reply:
On Android we can only make the whole message untranslatable, not just a word. In that case which one shall I mark non-traslatable? @rmcfadden3

Reply:
I would suggest just marking `IDS_ENABLE_SPEEDREADER_TITLE` as `translateable="false"`. This way we won't upload it for translations.

Reply:
Updated.